valhalla/jawn/src/middleware/auth.ts

```diff
@@ -33,6 +49,30 @@ export const authMiddleware = async (
       return;
     }
 
+    const rateLimit = await checkRateLimit(authParams.data?.organizationId);
+    if (rateLimit && !rateLimit.success) {
+      console.error("Rate limit exceeded", authParams.data?.organizationId);
```

Comment:
Can we leave console.error to more exceptional errors or else we will just get bombarded with these

Reply:
yeah